### ISSUE TITLE
Intermittent failures in 0.1.6

### ISSUE THREAD

**Issue by sbleon** [trailblazer/reform-rails#27]
This change https://github.com/trailblazer/reform-rails/commit/2e2a1ae25f48a9ec2385fa661ab6abb39241497f#diff-7039e69b74a923442bc1ec818227ae27L54 makes my specs fail with:
```
ArgumentError: wrong number of arguments (given 3, expected 2)
```

**Comment by fran-worley** [comment]
@sbleon you need to make sure that you are using the right version of Reform.

**Comment by sbleon** [comment]
I’m using reform 2.2.1 (the latest release). Is there something I should be
doing differently?

**Comment by ZombieHarvester** [comment]
same problem
```
    reform (2.2.1)
      disposable (>= 0.3.0, < 0.4.0)
      representable (>= 2.4.0, < 3.1.0)
      uber (~> 0.0.11)
    reform-rails (0.1.6)
      activemodel (>= 3.2)
      reform (>= 2.2.0)
```

**Comment by kapso** [comment]
+1 ...seeing the same issue.

**Comment by fran-worley** [comment]
Ahh see that commit probably shouldn't have got released as you'll need reform from master. 

I'll pull it out and stick it in a branch for now. Sorry guys!

**Comment by sbleon** [comment]
Thanks, Fran!

**Comment by sbleon** [comment]
This is fixed in `master` so I'll close this issue.

**Comment by sbleon** [comment]
Specifically, it was fixed by e8dea9ff159d238a622065581a3650f50bba33a7

**Comment by fran-worley** [comment]
0.1.7 now released and 0.1.6 yanked. Thanks for reporting guys!
